apps/andi/priv/static/js/app.js

```diff
@@ -0,0 +1,120 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
This appears to be a generated file, but it made me think about have we given thought to how we would TDD JavaScript inside of a phoenix app?